Documentation/contributing/development/dev_setup.rst

```diff
@@ -9,6 +9,19 @@
 Development Setup
 =================
 
+Dev Container
+~~~~~~~~~~~~~
+
+Cilium provides Dev Container configuration for  Visual Studio Code Remote Containers
```

Comment:
Is it worth linking to something here? When I read these paragraphs, I don't really know what Dev Container is or what I should do about this paragraph.

Reply:
Good point :100: